pkg/types/openstack/validation/machinepool.go

```diff
@@ -24,5 +26,29 @@ func ValidateMachinePool(_ *openstack.Platform, machinePool *openstack.MachinePo
 	default:
 		errs = append(errs, field.NotSupported(fldPath.Child("serverGroupPolicy"), machinePool.ServerGroupPolicy, validServerGroupPolicies))
 	}
+
```

Comment:
Innocent question but aren't we setting a guard to test if `featureSet` has `TechPreviewNoUpgrade` ? Or this is not TechPreview and directly stable?

I understand that API hasn't been changed in OpenShift API, however the user experience in the installer is changing a bit, so I wonder if the plan is to make these news params as Tech Preview or not.

I'm fine either way, I'm just making sure that this was discussed.

Reply:
It’s just below! Unless I’m missing something?

Reply:
Oops, didn't see it!